lib/linguist/heuristics.yml

```diff
@@ -216,6 +216,13 @@ disambiguations:
     pattern: '^<\?(?:php)?'
   - language: SourcePawn
     pattern: '^public\s+(?:SharedPlugin(?:\s+|:)__pl_\w+\s*=(?:\s*{)?|(?:void\s+)?__pl_\w+_SetNTVOptional\(\)(?:\s*{)?)'
+  - language: NASL
+    pattern:
+    - '^\s*include\s*\(\s*(?:"|'')[\\/\w\-\.:\s]+(?:"|'')\s*\)\s*;'
```

Comment:
Is it possible to `include` files without a `.nasl` or `.inc` extension? If not, it might be a good idea to add them to the regex to avoid matching files with `include("foo.php")` or what-have-you. You can use this:

~~~regex
- '^\s*include\s*\(\s*("|'')(?:(?!\1)[^\r\n])*\.(?:inc|nasl)\1\s*\)\s*;'
~~~

Reply:
Technically the extension can be anything but I never saw it with anything different than nasl or inc so I think it is better to restrict this regex to avoid false positives. Updated in latest commit.